esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -5938,22 +5936,22 @@ void CWsDfuEx::getFileMeta(StringBuffer &metaInfoStr, IDistributedFile &file, IU
      * Should be part of the same configuration setup.
      */
 #ifdef _USE_OPENSSL
-    if (!isEmptyString(keyPairName)) // without it, meta data is not encrypted
+    if (!keyPairName.isEmpty()) // without it, meta data is not encrypted
     {
         metaInfo->setProp("version", securityInfoVersion);
-        metaInfo->setProp("logicalFilename", file.queryLogicalName());
-        metaInfo->setProp("jobId", req.getJobId());
+        metaInfo->setProp("logicalFilename", fileName);
+        metaInfo->setProp("jobId", jobId);
         metaInfo->setProp("accessType", req.getAccessTypeAsString());
         StringBuffer userStr;
         if (user)
             metaInfo->setProp("user", user->getUserName(userStr).str());
-        metaInfo->setProp("keyPairName", keyPairName);
-        metaInfo->setProp("expiryTime", expiryTime);
+        metaInfo->setProp("keyPairName", keyPairName.str());
+        metaInfo->setProp("expiryTime", expiryTime.str());
```

Comment:
no need for these .str()'s - operator overload will handle it.

Reply:
ok